main.go

```diff
@@ -2077,10 +2077,11 @@ func parseGitConfigs(configsFlag string) ([]keyVal, error) {
 	ch := make(chan rune)
 	stop := make(chan bool)
 	go func() {
+	loopOverConfigFlags:
```

Comment:
yes, BUT....this whole function is borked.  We never send stop, so we leave a dangling goroutine.  I only vaguely recall this code, and frankly, I am not sure what I was thinking.  I think this can be much simpler.

Adding leak detection to the test proves it.

I'm going to close this and send a different one which doesn't leak.